### DISCUSSION TITLE
Adding to a rtmp stream

### DISCUSSION THREAD

**Discussion by varunsridharan** [ant-media/Ant-Media-Server#6053] [General]
I've accomplished the installation and configuration of the Ant Media Server Community Edition, effectively enabling a smooth live streaming experience. At present, I am actively transmitting video data to the server using the provided RTMP connection details.

My ongoing objective is to broaden the audience reach of the live stream by simultaneously publishing it to two separate RTMP servers. This endeavor includes intentionally introducing a delay of 1 to 2 minutes for one of the RTMP servers to receive and subsequently broadcast the stream.

Any assistance or guidance towards achieving this goal would be highly appreciated.

**Comment by Mohit-3196** [comment]
Hi @varunsridharan,
How are you?

I think that the simplest way would be to either start the stream and different points or to add the RTMP endpoint after a minute or two that you want to have.

I think other community members will comment if there is another easier way to achieve it. 

All the best!
